### PR TITLE
update for heroku deployment

### PR DESCRIPTION
**Pull request by Rostammahabadi**
Added config.js
added .env to .gitignore (will not have access to this)
updated database.js to account for 
updated .sequelizerc to account for change of file path to configuration

### CODE REVIEW

File: src/config/config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+module.exports = {
+  development: {
+    username: 'rostammahabadi',
+    password: null,
+    database: 'initiateco',
+    host: '127.0.0.1',
+    dialect: 'postgresql',
+    seederStorage: "sequelize"
+  },
+  production: {
```
Comment (muydanny) [inline src/config/config.js:12]: Great use of environment variables so our information isn't exposed. Bless.

File: .sequelizerc [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const path = require('path');
 
 module.exports = {
-  'config': path.resolve('src', 'config', 'database.json'),
-  'models-path': path.resolve('src', 'models'),
+  'config': path.resolve('src', 'config', 'config.js'),
```
Comment (muydanny) [inline .sequelizerc:4]: Great idea using a config.s instead of a database.json.

**Review by cwefso — APPROVED** [review]
This looks just magical

**Review by muydanny — APPROVED** [review]

File: src/config/config.js (hunk 1 above)
Comment (foleymichelle9) [reply src/config/config.js:12]: Very very nice!

**Comment by foleymichelle9** [comment]
Looks great! I definitely want to follow up with you guys and get a walkthrough!

**Pull request merged**